### PR TITLE
Admin cli fixes

### PR DESCRIPTION
**Pull request by jayrbolton**
This includes a backlog of bugfixes I've wanted to do after manually testing the admin CLI in CI a while ago. The piece pulled from Erik's PR was wrapping some workspace and object ids in an `int()` cast.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,4 @@
---extra-index-url https://pypi.anaconda.org/kbase/simple
-kbase_workspace_client==0.3.0
+kbase_workspace_client==0.0.1
```
Comment (jayrbolton) [inline requirements.txt:1]: Installing from a poetry build published to PyPI now instead of Anaconda

File: src/admin_tools/main.py [hunk 2]
```diff
@@ -47,6 +46,7 @@ def _get_upas(args):
     resp = requests.post(
         _ERR_SEARCH_URL,
         params={'scroll': '1m'},
+        headers={'Content-Type': 'application/json'},
```
Comment (jayrbolton) [inline src/admin_tools/main.py:49]: Elasticsearch requires this now

File: src/admin_tools/main.py [hunk 3]
```diff
@@ -93,13 +94,13 @@ def _reindex(args):
         raise ValueError("--ref value should be in the format '1/2', '1/2/3', or '1'")
     reindexing_obj = len(id_pieces) >= 2
     if reindexing_obj:
-        ev = {'evtype': 'INDEX_NONEXISTENT', 'wsid': id_pieces[0], 'objid': id_pieces[1]}
+        ev = {'evtype': 'INDEX_NONEXISTENT', 'wsid': int(id_pieces[0]), 'objid': int(id_pieces[1])}
```
Comment (jayrbolton) [inline src/admin_tools/main.py:97]: Wrapping all this in `int()` doesn't really affect anything in the stack, but seems like good practice.

File: src/admin_tools/main.py [hunk 4]
```diff
@@ -127,27 +128,28 @@ def _reindex_ws_type(args):
...
     #   - For each obj matching args.type, produce a reindex event
-    ws = WorkspaceClient(url=config()['kbase_endpoint'], token=config()['ws_token'])
+    ws_client = WorkspaceClient(url=config()['kbase_endpoint'], token=config()['ws_token'])
     evtype = 'INDEX_NONEXISTENT'
     if args.overwrite:
         evtype = 'REINDEX'
     for wsid in range(args.start, args.stop + 1):
+        wsid = int(wsid)
         try:
-            infos = ws.admin_req('listObjects', {'ids': [wsid]})
```
Comment (jayrbolton) [inline src/admin_tools/main.py:136]: This would hit a 10k limit. My mistake.

File: src/admin_tools/main.py [hunk 5]
```diff
@@ -127,27 +128,28 @@ def _reindex_ws_type(args):
...
-        except WorkspaceResponseError as err:
-            print(err.resp_data['error']['message'])
+            infos = ws_client.generate_obj_infos(wsid, admin=True)
```
Comment (jayrbolton) [inline src/admin_tools/main.py:138]: .generate_obj_infos does not hit the 10k limit and does repeated requests

**Comment by scanon** [comment]
LGTM

**Comment by jayrbolton** [comment]
Travis fails because kbase_workspace_client requires python 3.7 and travis only runs 3.6 by default. That package works fine with 3.6, just tested it, so I'll publish a patch version for that package that allows 3.6

**Comment by jayrbolton** [comment]
tests are ok now

**Pull request merged**